var/spack/repos/builtin/packages/hsakmt-roct/package.py

```diff
@@ -109,9 +109,12 @@ def check_install(self):
                 "-DLIBHSAKMT_PATH=" + hsakmt_path,
                 ".",
             ]
-            self.run_test(cmake_bin, cc_options)
+            cmake = self.spec["cmake"].command
+            cmake(*cc_options)
+            make = which("make")
             make()
             os.environ["LD_LIBRARY_PATH"] = hsakmt_path
             os.environ["BIN_DIR"] = os.getcwd()
-            self.run_test("scripts/run_kfdtest.sh")
+            scripts = which(join_path("scripts", "run_kfdtest.sh"))
+            scripts()
```

Comment:
This is using the name of the subdirectory, not the script:

```suggestion
            run_kfdtest = which(join_path("scripts", "run_kfdtest.sh"))
            run_kfdtest()
```